pkg/apis/ceph.rook.io/v1/resources.go

```diff
@@ -95,6 +97,11 @@ func GetLogCollectorResources(p ResourceSpec) v1.ResourceRequirements {
 	return p[ResourcesKeyLogCollector]
 }
 
+// GetRGWOpsLogResources returns the placement for the crash daemon
+func GetRGWOpsLogResources(p ResourceSpec) v1.ResourceRequirements {
```

Comment:
Latest Go/k8s best practices suggest using acronyms in Camel case (ACRONYM -> Acronym) for readability

```suggestion
// GetRgwOpsLogResources returns the placement for the crash daemon
func GetRgwOpsLogResources(p ResourceSpec) v1.ResourceRequirements {
```